lib/debug/server_dap.rb

```diff
@@ -465,9 +465,7 @@ def process
 
     ## called by the SESSION thread
 
-    def respond req, res
-      send_response(req, **res)
-    end
+    alias respond send_response
```

Comment:
I cannot accept this change because this change breaks existing codes.
```ruby
def send_response(req, **res)
  puts req, res
end

# def respond req, res
#   send_response(req, **res)
# end

alias respond send_response

def send_response(req, **res)
  puts req, res
end

respond({}, {a: 1}) # => wrong number of arguments (given 2, expected 1) (ArgumentError)
```

Reply:
No problem. Thank you for your contribution 👍